### ISSUE TITLE
App crashes consistently when opening a link

### ISSUE THREAD

**Issue by its-yourmom** [hariettem/Artemis#392]
**Describe the bug**
App crashes consistently when opening a link in the asklemmy sidebar.

**To Reproduce**
1. Go to Artemis, click search on the bottom.
2. Click on “asklemmy”
3. Tap the open sidebar
4. Scroll down to the ask Ouija link, click on “open in app”
5. See error. App crashes.

**Expected behavior**
For the link to open in the app, and for the app not to crash.

**Screenshots**
If applicable, add screenshots to help explain your problem. (Hopefully the screen recording will be attached 🤞🏻

**Smartphone (please complete the following information):**
 - Device: iPhone 11 Pro
 - OS: iOS 16.1.1
 - Version Version: 0.2.0

Update: 2308.13.16.17
(4910df5e-459c-4ba8-ac60-7f8c1aa7773a)}

Channel: preview

**Additional context**
I was able to reproduce this particular crash consistently. App crashed about 5 times before I started this ticket.

**Comment by its-yourmom** [comment]
https://cdn.discordapp.com/attachments/1125093022179278908/1141786209593606214/RPReplay_Final1692292589.mov
I hope this links to the screen recording I made of the issue on Discord in the beta squad…

**Comment by its-yourmom** [comment]
Idk what I’m doing here. I’m so sorry. 
https://discord.com/channels/1118996335119569036/1125093022179278908/1141786210021412944
Does this link work?

**Comment by RheingoldRiver** [comment]
yes it does, and as a note to readers the most convenient way to open it in the desktop client is to copy link and then paste it into some spam channel and then click it from there
